packages/api/controllers/hackathon_register_controller.go

```diff
@@ -164,13 +160,16 @@ func CompareTeamNames(passedTeamName string, storedTeamName string) bool {
 	return FormatTeamName(passedTeamName) == FormatTeamName(storedTeamName)
 }
 
-func GetTotalNumberOfParticipants() int64 {
+func GetTotalNumberOfParticipants(c *fiber.Ctx) error {
```

Comment:
You could probably pass the context of the /register endpoint to this one and just check the result

Reply:
Ok I will check